### ISSUE TITLE
Python test suite failure (eigenvalues)

### ISSUE THREAD

**Issue by 0u812** [sys-bio/roadrunner#137]
Eigenvalue test in `testing/test_data/oscli.rrtest` fails with the following output:
```
Starting Test on  /home/poltergeist/devel/install/roadrunner/site-packages/roadrunner/testing/test_data/oscli.rrtest
Successfully loaded model.

Notice: performing conserved moiety conversion
Notice: source document is level 2, version 1, converting to level 3, version 1 for Moiety Conservation Conversion
Check [Set Steady State Selection List]       PASS
Computing Steady State.  Distance to SteadyState: 2.44931911407e-10
Check [Get Steady State Selection List]       PASS
Check [Species Concentrations]                PASS
Computing Steady State.  Distance to SteadyState: 0.0
Check [Steady State Fluxes]                   PASS
Check [Get Reaction Rates]                    PASS
Check [Full Jacobian]                         PASS
Check [Reduced Jacobian]                      PASS
Check [Individual Eigenvalues]                *****FAIL*****
Check [Eigenvalue Matrix]                     PASS
Check [Reduced Eigenvalue Matrix]             PASS
Check [Stoichiometry Matrix]                  PASS
Check [Reduced Stoichiometry Matrix]          PASS
Check [Link Matrix]                           PASS
Check [Unscaled Elasticity Matrix]            PASS
Check [Unscaled Concentration Control Matrix] PASS
Check [Scaled Concentration Control Matrix]   PASS
Check [Unscaled Flux Control Matrix]          PASS
```

**Comment by hsauro** [comment]
Check species order, that's been an issue in the past.

Herbert

**Comment by 0u812** [comment]
Fixed in faadfd6a3c85c432976d95cff18484a803613d2a (had wrong issue #)
